t/sanity.t

```diff
@@ -1451,3 +1451,82 @@ healthcheck: peer 127\.0\.0\.1:12355 was checked to be not ok
 healthcheck: peer 127\.0\.0\.1:12356 was checked to be not ok
 healthcheck: peer 127\.0\.0\.1:12359 was checked to be not ok
 $/
+
+=== TEST 16: effect of no_timer (healthcheck options can be changed)
```

Comment:
style: we need three blank lines before this line. you are recommended to use the `reindex` tool to do it automatically:
https://github.com/openresty/openresty-devel-utils/blob/master/reindex

doc is still in the PR: https://github.com/openresty/openresty-devel-utils/pull/29/files#diff-04c6e90faac2675aa89e2176d2eec7d8R237

Reply:
done